more_itertools/tests/test_recipes.py

```diff
@@ -344,6 +344,50 @@ def test_combinatorics(self):
         )
 
 
+class UniquePowersetTests(TestCase):
```

Comment:
Could you add a test for what's expected with an empty iterable?

Reply:
done in [9eba55](https://github.com/root-11/more-itertools/commit/9eba5565496d59305c3578a4e242ad069373ba1b)